### DISCUSSION TITLE
getting MOVE redirects if I use the configuration endpoint for elasticache

### DISCUSSION THREAD

**Discussion by piccione99** [redis/go-redis#3039] [General]
My understanding is go-redis is redis OSS compatible, but it doesn't seem to work as expected.  I have a redis cluster set up and if I use the elasticache configuration endpoint in the Addrs option for ClusterClient, I constantly get "MOVE" redirects on set and it never saves.  There are currently 3 shards and the number of redirects is set to the default of 3 so I don't think it's a question of not enough redirects. 

If I instead modify the configuration to point to all the actual shard endpoints, it works.  Shouldn't the configuration file also work?  

If go-redis is not really redis OSS compatible can that be stated in the readme?

**Comment by monkey92t** [comment]
We're sorry for the trouble this has caused you. We have not tried using go-redis with ElastiCache clusters and have not indicated compatibility with ElastiCache in the README. If you could provide help to ensure compatibility, we would greatly appreciate it.
